### DISCUSSION TITLE
[@types/pdfmake] missing subset for TDocumentInformation

### DISCUSSION THREAD

**Discussion by tinohager** [DefinitelyTyped/DefinitelyTyped#71478] [Issues with a @types package]
> Object literal may only specify known properties, and 'subset' does not exist in type 'TDocumentDefinitions'.ts(2353)

The `subset` is missing
https://github.com/bpampuch/pdfmake/blob/master/examples/pdfa.js

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/pdfmake/interfaces.d.ts#L172

[a comment by an automated account is not shown]

**Comment by kryops** [comment]
PDF/A support was recently added to PDFmake: https://github.com/bpampuch/pdfmake/commit/ca6f874dc1c9c66146f7ce56352291b76482e12b

`tagged` and `displayTitle` were added in addition to `subset`, and should ideally be added to the type definitions as well.

**Reply by kryops** [reply]
Managed to find the time to open a PR: https://github.com/DefinitelyTyped/DefinitelyTyped/pull/71556
